spring-cloud-skipper-platform-cloudfoundry/src/main/java/org/springframework/cloud/skipper/deployer/cloudfoundry/CloudFoundryManifestApplicationDeployer.java

```diff
@@ -99,7 +99,7 @@ public ApplicationManifest getCFApplicationManifest(Release release) {
 					.disk(CloudFoundryApplicationManifestUtils.memoryInteger(manifest.getDiskQuota()))
 					.domains(manifest.getDomains() != null && !manifest.getDomains().isEmpty() ? manifest.getDomains()
 							: null)
-					.environmentVariables(new HashMap<>())
+					.environmentVariables(manifest.getEnv() != null ? manifest.getEnv() : null)
```

Comment:
@jvalkeal Have you tried passing the env via `properties` cmdline option?

Reply:
Yes `package install --package-name logcf --release-name mylogcf --platform-name cf-dev --package-version 1.0.0 --properties "spec.manifest.env={key1: value1, key2: value2}"` worked just fine.